src/Forms/TextareaField.php

```diff
@@ -174,6 +174,6 @@ public function Type()
      */
     public function ValueEntities()
     {
-        return htmlentities($this->Value() ?? '', ENT_COMPAT, 'UTF-8');
+        return htmlentities($this->Value() ?? '', ENT_COMPAT, 'UTF-8', false);
```

Comment:
This will break normal `TextareaField` implementations.

For example, set up your main `Page` class like so:

```php
use SilverStripe\CMS\Model\SiteTree;
use SilverStripe\Forms\TextareaField;

class Page extends SiteTree
{
    private static $db = [
        'JustSomeText' => 'Text',
    ];

    private static $has_one = [];

    public function getCMSFields()
    {
        $fields = parent::getCMSFields();
        $fields->addFieldsToTab('Root.Main', [TextareaField::create('JustSomeText')]);
        return $fields;
    }
}
```

Then add the following text to your new TextArea and save or publish:

> To escape an ampersand in HTML, use "&amp;"

You'll see that your text has changed to this:

> To escape an ampersand in HTML, use "&"

which is _not_ the text you originally entered.

Reply:
Sorry, could you clarify you expectations about this behaviour. Since, I get identical result for  `htmlentities($this->Value() ?? '', ENT_COMPAT, 'UTF-8', false);` and `htmlentities($this->Value() ?? '', ENT_COMPAT, 'UTF-8');`. DB also store the identical result.
In your example you have two identical string "To escape an ampersand in HTML, use "&"". Maybe, I do something wrong.

Thank you.

Reply:
Hah! Sorry, GitHub was encoding my `&ap;amp;` as `&` - I've had to double encode it here to show what I meant.

Reply:
This change still hasn't been made?

Reply:
Updated